public/fallback-H0EKsZXJpVaFw23tAwpN3.js

```
@@ -0,0 +1 @@
+(()=>{"use strict";self.fallback=async e=>"document"===e.destination?caches.match("/offline.html",{ignoreSearch:!0}):Response.error()})();
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)